Gemfile

```diff
@@ -4,52 +4,71 @@ source "https://rubygems.org"
 
 ruby RUBY_VERSION
 
-gem "decidim", git: "https://github.com/OpenSourcePolitics/decidim.git", branch: "0.18-merge"
+gem "decidim", git: "https://github.com/decidim/decidim.git", branch: "release/0.22-stable"
 # gem "decidim", path: "../decidim"
 # gem "decidim-map", path: "../decidim-map"
 
-# gem "decidim-conferences", git: "https://github.com/OpenSourcePolitics/decidim.git", branch: "0.16-stable"
-# gem "decidim-consultations", git: "https://github.com/OpenSourcePolitics/decidim.git", branch: "0.16-stable"
-# gem "decidim-initiatives", git: "https://github.com/OpenSourcePolitics/decidim.git", branch: "0.16-stable"
+# gem "decidim-conferences", git: "https://github.com/decidim/decidim.git", branch: "0.21-stable"
+# gem "decidim-consultations", git: "https://github.com/decidim/decidim.git", branch: "0.21-stable"
+# gem "decidim-initiatives", git: "https://github.com/decidim/decidim.git", branch: "0.21-stable"
 
 # gem "decidim-conferences", path: "../decidim"
 # gem "decidim-consultations", path: "../decidim"
 # gem "decidim-initiatives", path: "../decidim"
 
-gem "decidim-phone_authorization_handler", git: "https://github.com/OpenSourcePolitics/decidim-module_phone_authorization_handler"
-gem "decidim-term_customizer", git: "https://github.com/OpenSourcePolitics/decidim-module-term_customizer.git", branch: "0.18-stable"
+#gem "decidim-calendar", git: "https://github.com/alabs/decidim-module-calendar"
+#gem "decidim-cookies", git: "https://github.com/OpenSourcePolitics/decidim-module_cookies", branch: "feature/orejime"
+#gem "decidim-combined_budgeting", git: "https://github.com/mainio/decidim-module-combined_budgeting"
+#gem "decidim-decidim_awesome", git: "https://github.com/Platoniq/decidim-module-decidim_awesome"
+#gem "decidim-comparative_stats", git: "https://github.com/Platoniq/decidim-module-comparative_stats"
+#gem "decidim-direct_verifications", git: "https://github.com/Platoniq/decidim-verifications-direct_verifications"
+#gem "decidim-homepage_interactive_map", git: "https://github.com/OpenSourcePolitics/decidim-module-homepage_interactive_map.git"
+#gem "decidim-initiatives_no_signature_allowed", git: "https://github.com/OpenSourcePolitics/decidim-module-initiatives_nosignature_allowed.git"
+#gem "decidim-navigation_maps", git: "https://github.com/Platoniq/decidim-module-navigation_maps"
+
+gem "decidim-socio_demographic_authorization_handler", git: "https://github.com/MoretS/decidim-module-socio_demographic_authorization_handler", branch: "release/0.22-stable"
+
+gem "decidim-phone_authorization_handler", git: "https://github.com/MoretS/decidim-module_phone_authorization_handler", branch: "release/0.22-stable"
```

Comment:
Should we not use the OpenSourcePolitics' repository for the gem `phone_authorization_handler` ?

Reply:
Indeed ! Modification applied